packages/web-react/src/components/CheckboxField/__tests__/useCheckboxFieldStyleProps.test.ts

```diff
@@ -30,11 +31,25 @@ describe('useCheckboxFieldStyleProps', () => {
     expect(result.current.classProps.label).toBe('CheckboxField__label CheckboxField__label--hidden');
   });
 
-  it('should return field with error', () => {
-    const props = { validationState: 'error' } as SpiritCheckboxFieldProps;
+  it('should return field with danger', () => {
```

Comment:
Using `it.each` will be better than repeating the same code again. See https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
done